05_syntax_lab/03.py

```diff
@@ -1,7 +1,14 @@
-"""
-Write a program that randomizes a number
-and prints the sum total of its digits.
-For example if the number was: 2345
-The result should be: 14
-"""
-
+
+from random import randrange
+number = randrange(1000) # from 0 to 9 and 10 is not included
+print number
+x = number % 10
+y = (int(number) / 10) 
+z = y / 10
+f = number % 100
+w = f / 10
+m = int (w + x + z)
+print x
+print z
+print w
+print m
```

Comment:
That would be much better done in a loop instead of so many repeated statements.